modules/machine-translation/settings-preview-machine-translation.php

```diff
@@ -0,0 +1,39 @@
+<?php
+/**
+ * @package Polylang
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Class to advertize the Machine Translation module.
+ *
+ * @since 3.6
+ */
+class PLL_Settings_Preview_Machine_Translation extends PLL_Settings_Module {
+	/**
+	 * Stores the display order priority.
+	 *
+	 * @var int
+	 */
+	public $priority = 85;
+
+	/**
+	 * Constructor.
+	 *
+	 * @since 3.6
+	 *
+	 * @param PLL_Settings $polylang Polylang object.
+	 */
+	public function __construct( &$polylang ) {
+		parent::__construct(
+			$polylang,
+			array(
+				'module'        => 'machine_translation',
+				'title'         => __( 'Machine Translation', 'polylang' ),
+				'description'   => __( 'Allows linkage to an external translation solution.', 'polylang' ),
+				'active_option' => '',
+			)
+		);
+	}
```

Comment:
For previews like this one (for share slugs and translated slugs too), I suggest to merge current arguments with arguments coming from the child class:
```suggestion
	public function __construct( &$polylang, $args = array() ) {
		$defaults = array(
			'module'        => 'machine_translation',
			'title'         => __( 'Machine Translation', 'polylang' ),
			'description'   => __( 'Allows linkage to an external translation solution.', 'polylang' ),
			'active_option' => '',
		);

		parent::__construct( $polylang, array_merge( $default, $args ) );
	}
```
What do you think

Reply:
Done in [9732372](https://github.com/polylang/polylang/pull/1420/commits/97323722ce8c9f85ede58ff4d9bdc3bc31584b72).